### ISSUE TITLE
Synchronous Example

### ISSUE THREAD

**Issue by spamwax** [ogrebgr/android_volley_examples#12]
I was wondering if you could add a synchronous example to your set that uses `RequestFuture` and `future.get(4, TimeUnit.SECONDS)`
I followed the sample in the [source code](https://android.googlesource.com/platform/frameworks/volley/+/master/src/main/java/com/android/volley/toolbox/RequestFuture.java) of the RequestFuture but future.get() blocks first and then times out with no result!

Thanks

**Comment by ogrebgr** [comment]
Hi, did you tested your request if it executes properly without using Future, i.e. as normal request?

**Comment by spamwax** [comment]
@ogrebgr yes, it does work properly. Adding a breakpoint I can trace the code and see the result that's returned from the server. But it seems it's happening in a different thread and I can't get it to send the results back to the calling thread.
